x/proof/keeper/service.go

```diff
@@ -0,0 +1,27 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+)
+
+// getServiceCupr ensures that a service with the ServiceID of the given service
+// exists.
+// It returns the compute units per relay for the service with the given id.
+func (k Keeper) getServiceCupr(
```

Comment:
I think we should avoid abbreviations and acronyms to improve readability:

```suggestion
func (k Keeper) getServiceComputeUnitsPerRelay(
```